### PR TITLE
Add term lookup and navigation

### LINKED ISSUE #330: Navigate between terms
* Narrower, broader and related terms are clickable.
* Route properly so browser’s back button works for going back to the previous term.
* No other navigation controls (back to previous term, breadcrumbs etc.) are needed.
* Due date 14 Nov (Dag van de termen).

### LINKED ISSUE #331: Look up terms by URI
UI to be fleshed out in some more detail. How can we integrate search and lookup? Source selection only needs to be shown for search, not lookup. Try to find examples of search UIs that have dynamic inputs (term selection in our case).

### PR DESCRIPTION
**Pull request by ddeboer**
* Fix #331.﻿
* Fix #330.

### CODE REVIEW

**Review by sdevalk — COMMENTED** [review]
Great work! 🕺

Perhaps we can discuss some minor UI details in our next meeting? (Not important enough for this PR, though - looking good already!)

File: src/router.ts [hunk 1]
```diff
@@ -9,7 +10,12 @@ export const router = createRouter({
     {
       path: '/',
       name: 'home',
-      component: Home,
+      component: SearchForm,
+    },
+    {
+      path: '/term',
```
[the remark this answers is not in the document]
Comment (ddeboer) [reply src/router.ts:16]: One consideration here is that the lookup URL may become some kind of permalink: `https://termennetwerk.digitaalerfgoed.nl/term?uri=http://vocab.getty…` (matches both Dutch and English) vs `https://termennetwerk.digitaalerfgoed.nl/lookup?uri=http://vocab.getty…` (English).

File: src/components/Datasets.vue [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <template>
   <select
+    id="datasets"
```
[the remark this answers is not in the document]
Comment (ddeboer) [reply src/components/Datasets.vue:3]: Sure, but that makes it slightly harder to differentiate the inputs (and the form a bit more boring).

File: src/components/SearchResults.vue [hunk 3]
```diff
@@ -0,0 +1,95 @@
+<template>
+  <div v-if="q && data">
+    <div
```
[the remark this answers is not in the document]
Comment (ddeboer) [reply src/components/SearchResults.vue:3]: Will do!

File: src/router.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ddeboer) [reply src/router.ts:16]: Decided to use `/lookup` (so we can add `/search` later, if needed).

**Review by sdevalk — APPROVED** [review]
Perfect!

**Pull request merged**